Parsers/ASimFileEvent/Parsers/vimFileEventMicrosoftWindowsEvents.yaml

```diff
@@ -13,12 +13,48 @@ References:
 - Title: ASIM
   Link: https://aka.ms/AboutASIM
 Description: |
-  This ASIM parser supports normalizing Microsoft Windows Events (WindowsEvent and SecurityEvent tables) to the ASIM File Event normalized schema. Event IDs which are parsed as part of this parser: 4663
+  This ASIM parser supports normalizing Microsoft Windows Events (WindowsEvent and SecurityEvent tables) to the ASIM File Event normalized schema. Event IDs which are parsed as part of this parser: 4663, 5140
 ParserName: vimFileEventMicrosoftWindowsEvents
 ParserQuery: |
-  let Parser=()
-  {
-  let EventTypeLookup = datatable (AccessMask:string,EventType:string)
+let Parser=()
+{  
+  let 5140AccessMaskLookup = datatable (AccessMask:string,EventType:string)
+  [
+      "0x1", "ReadData"
+      , "0x2", "WriteData"
+      , "0x4", "AppendData"
+      , "0x8", "ReadEA"
+      , "0x10", "WriteEA"
+      , "0x20", "Execute/Traverse"
+      , "0x40", "DeleteChild"
+      , "0x80", "ReadAttributes"
+      , "0x100", "WriteAttributes"
+      , "0x10000", "Delete"
+      , "0x20000", "Read_Control"
+      , "0x40000", "Write_Dac"
+      , "0x80000", "Write_Owner"
+      , "0x100000", "Synchronize"
+      , "0x1000000", "Access_Sys_Sec"
+  ];
```

Comment:
EventType is not standard.

Reply:
That I see ... but as described earlier ... the existing parser also doesn't set valid eventtype fields ?